app/services/audit/AuditService.scala

```diff
@@ -43,8 +42,8 @@ class AuditService @Inject()(auditConnector: DefaultAuditConnector,
     )
 
   private def generateTags(session: Session, hc: HeaderCarrier): Map[String, String] =
-    hc.headers.toMap ++
-      hc.headers.toMap ++
+    hc.otherHeaders.toMap ++
```

Comment:
Does `otherHeaders` contain all the headers needed here?

Reply:
I believe headers was split into `otherHeaders` (those built implicitly) and `extraHeaders` (those added to a request explicitly) so I believe `otherHeaders` is suitable here. Please correct me if I'm wrong though.